docs/blocks/blocks.md

```diff
@@ -8,18 +8,30 @@ Creating a Block
 
 ### Basic Blocks
 
-For simple blocks, which need no special functionality (think cobblestone, wooden planks, etc.), a custom class is not necessary. You can create a block by instantiating the `Block` class with a `Block.Properties` object. This `Block.Properties` object can be made using `Block.Properties#create` and it can be customised by calling its methods. For instance:
+If the block you are creating has no special functionality (think cobblestone, wooden planks), it is not necessary to create a new class for your block. You can simply instantiate the `Block` class. When doing this you will need a `Block.Properties` to pass to the constructor. This is how we set the properties of a block in 1.15+. Here's an exmaple:
 
-- `hardnessAndResistance` - The hardness controls the time it takes to break the block. It is an arbitrary value. For reference, stone has a hardness of 1.5, and dirt 0.5. If the block should be unbreakable a hardness of -1.0 should be used, see the definition of `bedrock` as an example. The resistance controls the explosion resistance of the block. For reference, stone has a resistance of 6.0, and dirt 0.5.
-- `sound` - Controls the sound the block makes when it is punched, broken, or placed. Requires a `SoundType` argument, see the [sounds][] page for more details.
-- `lightValue` - Controls the light emission of the block. Takes a value from zero to fifteen.
-- `slipperiness` - Controls how slippery the block is. For reference, ice has a slipperiness of 0.98.
+```java
+new Block(Block.Properties.from(Blocks.COBBLESTONE).lightValue(15)).setRegistryName("mymod:myblock");
+```
 
-All these methods are *chainable* which means you can call them in series. See the `Blocks` class for examples of this.
+See `Block.registerBlocks` for more examples on how to create simple blocks.
 
 !!! Note
 
-    Blocks have no setter for their `ItemGroup`. (formerly Creative Tab) This has been moved to the `BlockItem` and is now its responsibility. Furthermore, there is no setter for translation key as it is now generated from the registry name.
+    Blocks have no setter for Item Group (formerly Creative Tab). This has been moved to the ItemBlock, and is now its responsibility. Furthermore, there is no setter for translation key (this is generated based on registry name now).
+
+### `Block.Properties`
+To set the properties of the block, you need to use a `Block.Properties`. There are a couple of options when creating a one. You can use `Block.Properties.from` to copy the properties of an existing block or `Block.Properties.create` to create a new one.
+
+The `Block.Properties` has the following setters:
+
+  - `doesNotBlockMovement` - Makes it so the block does not block movement. Example Usage: plants.
```

Comment:
This is restating the name of the variable. Something like `allows the entity to pass through the block with no collision` explains the property better. Also, inconsistent formatting on the example usage compared to others.